src/DayHospital.java

```diff
@@ -73,37 +91,65 @@ public static void main(String[] args) {
 				++dayNo;
 			}
 			
+			/**
+			 * If the input is equal to a/A, an audit simulation is run on the current kennel status and vet status.
+			 * The function auditKennels() is invoked.
+			 * The system displays the animals for the Dog Kennels, General Kennels and Cat Kennels with the following information:
+			 *  - Name
+			 *  - Breed
+			 *  - Stay Duration
+			 *  - Assigned Vet
+			 *  - Check in Clerk
+			 * The system displays the animals for each of the vets with the following information:
+			 *  - Name
+			 *  - Breed
+			 *  - Fur Type
+			 *  - Weight
+			 * The animals contained in the kennels array should be the same as what is assigned to the vets.
+			 * Note that vets are assigned animals by preference of least number of animals per vet first, so the kennel arrays and vet arrays will not correspond to each other.
+			 * Rather the unique animal instances in the kennels array should be same as the unique animal instances in the vet arrays.
+			 */
 			if(temp.equalsIgnoreCase(audit)) {
 				petInterCare.auditKennels();
 			}
 			
+			/**
+			 * If the input is equal to q/R, the main program is terminated.
+			 */
 			if(temp.equalsIgnoreCase(quit)) { //should end application
 				System.out.println("You have left the Vet");
-	//			System.exit(0);
+				input.close();
+	//			System.exit(0); this is not required with the while loop implemented
 			}
 		}
 	}
 
+	/**
+	 * Returns a random float between the integers 3 and 20
+	 * @return	A random float, weight
+	 */
 	public static float getRandomWeight() {
 		float weight = (float)(Math.random()*20+3);
 		return weight;
-		
 	}
 	
+	/**
+	 * Returns a random float between the floats 1.1 and 0.2
```

Comment:
This is incorrect, and the wording is weird. 0.2 and 1.3. (by wording is weird I mean that you would normally start with the lowest value)

Reply:
changed to "floats between 0.2 and 1.3"